addons/launcher/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "launcher",
+    "summary": "Launches threads at server start time.",
+    "description": "Launches threads at server start time.",
+    "author": "Unipart Digital Team",
+    "website": "http://www.unipart.io",
```

Comment:
I think would be better to refer manifests files where Miquel created last 2 MR-s to check author and website

Reply:
Addressed in 60b0876edbb